spec/controllers/sufia/admin/admin_sets_controller_spec.rb

```diff
@@ -43,25 +43,27 @@
     end
 
     describe "#create" do
+      let(:service) { instance_double(Sufia::AdminSetService) }
+      before do
+        allow(Sufia::AdminSetService).to receive(:new)
```

Comment:
I'm following the pattern layed out here: https://blog.engineyard.com/2014/keeping-your-rails-controllers-dry-with-services

I don't think the law of demeter is applicable, because neither option exposes more information than the other.